ldk/javascript/packages/create-loop/index.js

```diff
@@ -0,0 +1,239 @@
+      }
+
+      return `${filename}${fileExtension}`;
+    };
+
+    // Target path: the project being created
+    const targetBasePath = `./${projectName}`;
+    // Source path: this script's template files
+    const sourceBasePath = path.join(__dirname, 'templates');
+
+    // Creating base template files from /template
+    const baseTemplateFiles = await fs.readdir(sourceBasePath);
+    const baseTemplateSquirrellyFiles = baseTemplateFiles.filter((filename) =>
+      filename.endsWith('.squirrelly'),
+    );
+
+    await fs.mkdir(targetBasePath);
+
+    baseTemplateSquirrellyFiles.forEach((squirrellyFilename) => {
+      const renderedFilename = squirrellyFilename.replace('.squirrelly', '');
+      renderTemplate(
+        path.join(sourceBasePath, squirrellyFilename),
+        path.join(targetBasePath, renderedFilename),
+      );
+    });
+
+    // Source for 'src' directory (./templates/src)
+    const sourceSrcPath = path.join(sourceBasePath, 'src');
+    // Creating base template source files from /template/src
+    const baseTemplateSrcFiles = await fs.readdir(sourceSrcPath);
+    const baseTemplateSrcSquirrellyFiles = baseTemplateSrcFiles.filter((filename) =>
+      filename.endsWith('.squirrelly'),
+    );
+
+    const targetSrcPath = path.join(targetBasePath, 'src');
+    await fs.mkdir(targetSrcPath);
+
+    baseTemplateSrcSquirrellyFiles.forEach((squirrellyFilename) => {
+      let renderedFilename = squirrellyFilename.replace('.squirrelly', '');
+      if (renderedFilename.endsWith('.ts')) {
+        renderedFilename = filenameWithExtension(renderedFilename);
+      }
+      renderTemplate(
+        path.join(sourceSrcPath, squirrellyFilename),
+        path.join(targetSrcPath, renderedFilename),
+      );
+    });
+
+    const sourceAptitudesPath = path.join(sourceSrcPath, 'aptitudes');
+    const targetAptitudesPath = path.join(targetSrcPath, 'aptitudes');
+    await fs.mkdir(targetAptitudesPath);
+
+    const sourceWhispersPath = path.join(sourceSrcPath, 'whispers');
+    const targetWhispersPath = path.join(targetSrcPath, 'whispers');
+    await fs.mkdir(targetWhispersPath);
+
+    // "search" aptitude create "ui" templates
+    aptitudes.forEach(async (aptitude) => {
+      let aptitudeFilename;
+      let whisperFilename = `${aptitude}Whisper`;
+      // Change first letter to uppercase, e.g. ClipboardWhisper
+      whisperFilename[0] = whisperFilename[0].toUpperCase();
+
+      switch (aptitude) {
+        case 'clipboard':
+          aptitudeFilename = 'clipboardListener';
+          break;
+        case 'filesystem':
+          aptitudeFilename = 'filesystemExample';
+          break;
+        case 'keyboard':
+          aptitudeFilename = 'keyboardListener';
+          break;
+        case 'network':
+          aptitudeFilename = 'networkExample';
+          break;
+        case 'search':
+          aptitudeFilename = 'searchListener';
+          whisperFilename = 'UiWhisper';
```

Comment:
Other than this guy, the rest of it makes it feel like this should be a simple object where the keys are the aptitude and the value is the name.